package.json

```diff
@@ -27,13 +27,13 @@
   "license": "Apache-2.0",
   "dependencies": {
     "bpk-tokens": "^36.0.0",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
-    "eslint": "^5.13.0",
-    "eslint_d": "^8.0.0",
```

Comment:
eslint_d was a very old install, and not one that we seem to need any more - for simplicity remove it if we can do without it.

https://github.com/mantoni/eslint_d.js